numpy/linalg/linalg.py

```diff
@@ -1300,19 +1300,21 @@ def svd(a, full_matrices=True, compute_uv=True):
     """
     Singular Value Decomposition.
 
-    Factors the matrix `a` as ``u * np.diag(s) * v``, where `u` and `v`
-    are unitary and `s` is a 1-d array of `a`'s singular values.
+    Factors the matrix `a` as ``u * np.diag(s) * vh``. When `a` is a 2D array,
+    `u` and `vh` are 2D unitary arrays and `s` is a 1D array of `a`'s singular
+    values.
 
     Parameters
     ----------
     a : (..., M, N) array_like
-        A real or complex matrix of shape (`M`, `N`) .
+        A real or complex matrix of shape (..., `M`, `N`) .
     full_matrices : bool, optional
-        If True (default), `u` and `v` have the shapes (`M`, `M`) and
-        (`N`, `N`), respectively.  Otherwise, the shapes are (`M`, `K`)
-        and (`K`, `N`), respectively, where `K` = min(`M`, `N`).
+        If True (default), `u` and `v` have the shapes (..., `M`, `M`) and
```

Comment:
Should this also be vh?

Reply:
Indeed. I'll double check for similar mistakes